chaosaws/ec2/actions.py

```diff
@@ -185,15 +186,23 @@ def get_instance_type_by_id(instance_ids: List[str],
     return get_instance_type_from_response(res)
 
 
-def stop_instances_any_type(instance_types: dict, force, client: boto3.client):
+def stop_instances_any_type(instance_types: dict = None,
```

Comment:
You are missing the return statement I think.

Reply:
thanks for noticing this